### DISCUSSION TITLE
ㄴㅇㄹㅇㄴㅁㄹ

### DISCUSSION THREAD

**Discussion by wngks1232** [storycraft/node-kakao#736] [Q&A] [answered]
맨션 보내기 할때 `@닉네임` 이런식이 아니고 맨션 대상자가 안보이게 보내는건 어떻게 하나요?

**Comment by wngks1232** [comment, marked as answer]
자문자답~!!

**Reply by tadashi74** [reply]
어떻게 하는지 알려주실 수 있나요 😮

**Reply by wngks1232** [reply]
네 node_modules\node-kakao\dist\chat\content\mention.js 파일에서 밑에 chat.text += `@${this.user.nickname}` 이소스를 chat.text += ``
요렇게 바꾸세요
